docs/guide/testing.md

```diff
@@ -31,9 +31,6 @@ _Note: `assert_selector` only matches on visible elements by default. To match o
 Since 2.56.0
 {: .label }
 
-Experimental
```

Comment:
Just double checking, we intentionally removed the experimental flags here?

Reply:
Yup 👍🏻